src/site/content/en/angular/faster-change-detection/index.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: "Optimize Angular's change detection"
+subhead: |
+  Faster change detection for better user experience.
+hero: change-detection.jpg
+alt: Different light bulb
+date: 2019-07-09
+description: |
+  Optimize the change detection of your app to make it more responsive and faster to use.
+authors:
+  - mgechev
+tags:
+  - angular
+---
+
+Angular runs its [change detection mechanism](https://angular.io/api/core/ChangeDetectorRef) periodically so that changes to the data model are reflected in an app's view. Change detection can be triggered either manually or through an asynchronous event (for example, a user interaction or an XHR completion).
+
+Change detection is a powerful tool, but if it's run very often, it can trigger a lot of computations and block the main browser thread.
+
+In this post you'll learn how to control and optimize the change detection mechanism by skipping parts of your application and running change detection only when necessary.
+
+
+## Inside Angular's change detection
+
+To understand how Angular's change detection works, let's look at a sample app!
+
+_You can find the code for the app in [this GitHub repository](https://github.com/mgechev/change-detection-web-dev)._
+
+
+![Sample application](app.png "Sample application")
+
+The app lists employees from two departments in a company—sales and R&D—and has two components:
+
+* `AppComponent`, which is the root component of the app, and
+* Two instances of `EmployeeListComponent`, one for sales and one for R&D.
+
```

Comment:
```suggestion

![Sample application](app.png "Sample application")

```
Sorry I didn't catch this before: we usually put the explanation _before_ the image so readers know what they're looking at.